tests/test_BaseSolver.py

```diff
@@ -1,15 +1,20 @@
+try:
+    from mpi4py import MPI
+except ImportError:
+    MPI = None
+
 import unittest
 from baseclasses import BaseSolver
 from baseclasses.utils import Error
 
 
 class SOLVER(BaseSolver):
-    def __init__(self, name, options=None, *args, **kwargs):
+    def __init__(self, name, options={}, comm=None, *args, **kwargs):
```

Comment:
I don't believe `args` and `kwargs` are used so those can probably be removed.

Reply:
Ok, will remove